### ISSUE TITLE
JSON skjema til C# klasser

### ISSUE THREAD

**Issue by oyei** [ks-no/fiks-plan-client-dotnet#21]
Jobbes det med å få skjemaene kompatible med opplegget i https://github.com/ks-no/fiks-plan-models-dotnet?

F.eks. i no.ks.fiks.gi.plan.v2.innsyn.arealplan.hent.schema.json får NasjonalArealplanId.AdministrativEnhet en uspesifisert type. 

Mulig noe som mangler i no.ks.fiks.gi.plan.v2.innsyn.dispensasjoner.resultat.schema.json også. Her et forsøk med nswag:

![image](https://user-images.githubusercontent.com/102580356/173541128-1a0b17af-42fd-45f7-bad8-b447f89e5b44.png)

**Comment by jarleborsheim** [comment]
Ja, jeg møtte på et liknende problem da jeg skulle generere kode fra json schema i dette repoet.
Vi har nå flyttet schemas til https://github.com/ks-no/fiks-plan-specification for å kunne lage en bygg pipeline som gjør at vi kan forhåpentligvis auto-generere nye C# modeller (og andre språk hvis vi ønsker) og levere det som nuget pakke. 
I den sammenheng fikk jeg problemer også og gjorde noen fikser i skjemaene slik at jeg i hvert fall fikk generert koden.
Så prøv gjerne på nytt med skjema fra det nye repoet.

**Comment by oyei** [comment]
Nevnte nuget-pakke er av interesse. Er den tilgjengelig nå?

**Comment by jarleborsheim** [comment]
Dessverre har vi ikke fått den ferdig enda. Vi håper å få den klar snart. Følg gjerne med på Slack kanalene  #fiks-integrasjon og #fiks-protokoller til KS hvor vi kommer med oppdateringer om disse tingene.
